### PR TITLE
Fix lint issues for MCF Browser

### PR DESCRIPTION
**Pull request by cwcrystal8**
This PR contains formatting changes after switching the linter over to a typescript linter. Also removes duplicates of FileEntryStateType and FileEntryPropType.

### CODE REVIEW

File: mcf-browser/src/FileEntry.tsx [hunk 1]
```diff
@@ -34,41 +34,7 @@ interface FileEntryPropType {
    * Sets whether the dropdown on the home page for additonal file entries
    * should be displayed.
    */
-  toggle: Function;
-}
-
```
Comment (pradh) [inline mcf-browser/src/FileEntry.tsx:39]: I forget what @beets wanted to do for this.  Was she okay deleting?

**Review by pradh — APPROVED** [review]

File: mcf-browser/src/FileEntry.tsx (hunk 1 above)
Comment (cwcrystal8) [reply mcf-browser/src/FileEntry.tsx:39]: Did she mention something about it in a previous PR?

Comment (pradh) [reply mcf-browser/src/FileEntry.tsx:39]: Thought you asked a question in the meeting and she said something. Or was it about something else?  In any case, please wait for Jennifer to take a look before merging...

Comment (cwcrystal8) [reply mcf-browser/src/FileEntry.tsx:39]: Sounds good

**Review by chejennifer — APPROVED** [review]
thanks for the lint fixes!

**Pull request merged**